projects/library/search/src/search.component.html

```diff
@@ -13,10 +13,12 @@
     [label]="inputLabel"
     [prefixIcon]="icon"
     [isClearable]="userCanClear"
+    [isDisabled]="isDisabled"
```

Comment:
heh literally just hit this 2 days ago 🥳

Reply:
It's funny that how the old specs all passed, but this line was never added in the template 😂